core/replicated_map.go

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core
+
+import (
+	. "github.com/hazelcast/hazelcast-go-client/serialization"
+	"time"
+)
+
+// ReplicatedMap is a map-like data structure with weak consistency
+// and values locally stored on every node of the cluster.
+//
```

Comment:
There shouldnt be any space here otherwise it wont be generated with goDoc

Reply:
No, it generates. The space is used for new paragraph.